large-borrows-governance/src/agent.spec.js

```diff
@@ -244,15 +270,17 @@ describe('handleTransaction', () => {
 
   it('returns two findings when a borrow event is emitted from the cCOMP token contract and the address that borrowed COMP exceeds the proposal and vote quorum thresholds', async () => {
     const {
-      minAmountCOMP: proposalMinCOMP, type: proposalType, severity: proposalSeverity,
+      type: proposalType, severity: proposalSeverity,
     } = borrowLevels.proposal;
     const {
-      minAmountCOMP: votingMinCOMP, type: votingQuorumType, severity: votingQuorumSeverity,
+      type: votingQuorumType, severity: votingQuorumSeverity,
     } = borrowLevels.votingQuorum;
 
     // set the balanceOf to a value that is greater than the minimum COMP threshold for the proposal
     // and vote quorum governance interactions
-    mockERC20Contract.balanceOf.mockResolvedValue(votingMinCOMP + 1);
+    // Mock: minAmountCOMP + 1, and then with the rest of the decimals added back on.
+    mockERC20Contract.balanceOf.mockResolvedValue(BigNumber.sum(minQuorumVotes, 1)
+      .multipliedBy(decimals));
```

Comment:
```suggestion
    // Mock: minAmountCOMP + 1, and then with the rest of the decimals added back on.
    const currCOMPOwned = BigNumber.sum(minQuorumVotes, 1);
    mockERC20Contract.balanceOf.mockResolvedValue(currCOMPOwned.multipliedBy(decimals));
```

Reply:
fixed in commit 47e475a088c41501b9359dff228780c15bb0a43b